.circleci/config.yml

```diff
@@ -8,8 +8,8 @@ orbs:
   ruby: circleci/ruby@2.1.1
   browser-tools: circleci/browser-tools@1.4.7
   snyk: snyk/snyk@1.1.2
-  aws-cli: circleci/aws-cli@4.1.3
-  aws-ecr: circleci/aws-ecr@9.0.2
```

Comment:
??? this reverts a required change to avoid end of life on the linux builds...

Reply:
I must have been working from a local version of the file which had the old values

Reply:
will look at what's in main and update